package.json

```diff
@@ -18,40 +17,37 @@
     "test": "jest",
     "test:watch": "NODE_ENV=test jest --watch",
     "update-snapshots": "NODE_ENV=test jest -u",
-    "lint":
-      "eslint tools/ examples/**/src/ packages/**/src/ --ext .js --ext .jsx -f table && gulp lint --env=test --theme"
+    "lint": "eslint tools/ examples/**/src/ packages/**/src/ --ext .js --ext .jsx -f table && gulp lint --env=test --theme"
```

Comment:
I will pay so much money to whoever figures out how to get Prettier to consistently format this file.

Reply:
Now that you mention it, I've noticed consistency problems in `package.json`'s too